ambry-clustermap/src/main/java/com/github/ambry/clustermap/HelixFactory.java

```diff
@@ -28,7 +30,9 @@
  */
 public class HelixFactory {
   private static final Logger LOGGER = LoggerFactory.getLogger(HelixFactory.class);
-  Map<ManagerKey, HelixManager> helixManagers = new ConcurrentHashMap<>();
+  // exposed for use in testing
+  final Map<ManagerKey, HelixManager> helixManagers = new ConcurrentHashMap<>();
```

Comment:
I see this map is only used by `HelixClusterManagerTest.java`. If so, can we move this map to `MockHelixManagerFactory` in that file?

Reply:
It is actually accessed in the getZkHelixManager method, but I was able to make it private and add a method that can be overridden for the test impl, which seems to be cleaner.